_includes/comment-form.html

```diff
@@ -25,7 +25,7 @@ <h5 id="form-title">Add a comment</h5>
 
         <div class="my-2">
             <label id="captcha-label" for="commentbox-captcha">Human verification</label><br />
-            <div class="g-recaptcha" data-sitekey="{{ site.reCaptcha.siteKey }}"></div>
+            <div id="captcha__comment__form" data-sitekey="{{ site.reCaptcha.siteKey }}"></div>
```

Comment:
It would be better to name it `captcha-holder` or something like that. Use `-` for differentiating words :)

Reply:
done